projects/angular-cld/src/lib/cloudinary-placeholder.component.ts

```diff
@@ -44,9 +45,9 @@ export class CloudinaryPlaceHolder implements AfterContentChecked {
 
   getPlaceholderImage() {
     if (this.type === 'predominant-color' && this.itemHeight && this.itemWidth) {
-      return this.cloudinary.url(this.publicId, {transformation: [this.options], placeholder: 'predominant-color-pixel' || true});
+      return this.cloudinary.url(this.publicId, {transformation: [this.options], placeholder: 'predominant-color-pixel' || true, ...SDKAnalyticsConstants});
```

Comment:
Add what in one place? placeholder is true or is not null bc it has a type

Reply:
I'm thinking about one place that prepare the options with the SDKAnalytics data

Reply:
I could make a util function that is called before we call cloudinary.url but it would be the same thing as sending in the options as we did above.

Reply:
Nope. now you use consts, if we ever need to change some of the values dynamically, you can't do it without changing all the places. 
Putting it in a single place will allow changing in one place only